hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/cluster/SparkExecuteClusteringCommitActionExecutor.java

```diff
@@ -50,6 +55,13 @@ public SparkExecuteClusteringCommitActionExecutor(HoodieEngineContext context,
 
   @Override
   public HoodieWriteMetadata<HoodieData<WriteStatus>> execute() {
+    if (config.isTableManagerEnabled() && config.getTableManagerConfig().getTableManagerActions().contains(ActionType.replacecommit.name())) {
+      LOG.warn("Clustering delegate to table management service, do not cluster for client!");
```

Comment:
Can we avoid invoking this code path from the client level, rather than returning an empty writeMetadata object

Reply:
Sure, will update it.